lib/vagrant.pm

```diff
@@ -33,16 +43,14 @@ sub run_vagrant_cmd {
 
     my $logfile = 'vagrant_cmd.log';
     local $@;
-    my $ret = eval {
-        return script_run("VAGRANT_LOG=DEBUG vagrant $cmd 2> $logfile", %args);
-    };
+    my $ret = script_run("VAGRANT_LOG=DEBUG vagrant $cmd 2> $logfile", %args);
+
     return undef if $ret == 0;
-    if (!$ret) {
-        upload_logs($logfile);
 
-        if ($@) {
-            die $@;
-        }
-        die "'vagrant $cmd' failed with $ret";
+    upload_logs($logfile);
+
+    if ($@) {
```

Comment:
The eval is gone so this won't work anymore either

Reply:
removed